packages/oauth/src/index.spec.js

```diff
@@ -417,8 +417,8 @@ describe('OAuth', async () => {
       assert.isTrue(sent);
     });
 
-    it('should call the failure callback due to missing code query parameter on the URL', async () => {
```

Comment:
I've essentially removed this test as it is line-for-line exactly the same as the previous test. While the intention of the two tests are different, because the callback handler needs _one of_ a `code` _or_ a `state` parameter returned, the result ends up a little off.

Alternatively, we could rewrite these tests a little to check that the callback handler passes if _either_ `state` _or_ `code` is provided.

Reply:
I've got a [branch open](https://github.com/slackapi/node-slack-sdk/pull/1329) with some updates to have these tests properly test `state` and `code` and throw different errors in each case. Related to making `state` optional depending on whether `stateValidation` flag is true/false.

Reply:
Ooooo nice!

Reply:
Perhaps just add your test separately for now, since it's testing something unrelated to either `state` or `code`?

Reply:
Sounds good, just pushed it back up!